package/upgrade/lib.sh

```diff
@@ -317,3 +317,49 @@ shutdown_all_vms()
       virtctl stop $name -n $namespace
     done
 }
+
+wait_for_fleet_bundles()
+{
+  # wait for the changes to be applied to bundle object first
+  # fleet-agent takes time to further apply changes to downstream deployments
+  sleep 10
+
+  # loop wait upto 3 minutes by default, do not block even when some bundles are not ready
+  # that needs manual operation
+  if [ -z $1 ]; then
+    local loop_cnt=18
+  else
+    local loop_cnt=$1
+  fi
```

Comment:
local loop_cnt=${1:-18}
```suggestion
local loop_cnt=${1:-18}
```